### ISSUE TITLE
utils.format_timestamp fails with daylight savings

### ISSUE THREAD

**Issue by mr-cal** [canonical/charmcraft#830]
### Issue
`utils.format_timestamp()` does not work in timezones with daylight savings.

### Reproducer
```
export TZ='America/Argentina/Buenos_Aires'
./run_tests tests/test_utils.py -k timestamp # passes

export TZ='America/Chicago'
./run_tests tests/test_utils.py -k timestamp # fails
```

**Comment by lengau** [comment]
@mr-cal I'm not sure whether this has been fixed, but I can't reproduce it on my own machine. Can you check?

**Comment by mr-cal** [comment]
Works for me and looks like it was fixed in #927
